docs/monorepo.md

```diff
@@ -86,6 +87,22 @@ To run one package's tests:
 
 `npm run test-packages [ test file pattern ]`
 
+## Building packages
+
+Packages are automatically build on `npm start` unless they're in `/apps` directory.
```

Comment:
```suggestion
Packages will have their `prepare` scripts run automatically on `npm install`.

Independent apps can use `build` instead of `prepare` so avoid unnecessary builds on `npm install`.
```